chats/views.py

```diff
@@ -3,13 +3,20 @@
 # Create your views here.
 from django.shortcuts import render, redirect, get_object_or_404
 from django.contrib.auth.decorators import login_required
-from .models import *
+from .models import ChatGroup
 from .forms import ChatmessageCreateForm
 
+def chat_view_home(request):
```

Comment:
```suggestion
def chat_view_home(request):
```
這裡既然是在 `views.py` 中，那就可以命名為 `chat_home` 就好，看起來比較簡潔

Reply:
收到